plugin-openshift/che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/importapp/ImportApplicationPresenter.java

```diff
@@ -243,6 +246,24 @@ public void apply(List<Project> result) throws OperationException {
     }
 
     /**
+     * Load Che projects.
+     */
+    private void loadCheProjects() {
+        projectServiceClient.getProjects(false).then(new Operation<List<ProjectDescriptor>>() {
+            @Override
+            public void apply(List<ProjectDescriptor> result) throws OperationException {
+                cheProjects = FluentIterable.from(result).transform(
+                        new com.google.common.base.Function<ProjectDescriptor, String>() {
+                            @Override
+                            public String apply(ProjectDescriptor input) {
+                                return input.getName();
+                            }
+                        }).toList();
```

Comment:
I think 
`cheProjects = new ArraysList<>(); // or cheProjects.clear(); but you should not use Collections.emptyList(); for init this variable`
`for (ProjectDescriptor project : result) {`
`    cheProjects.add(project.getName());`
`}`
is better here. WDYT?